std_msgs/Header header
uint8 node_type
uint8[] payload
